### DISCUSSION TITLE
Config Error in Default Configuration : Noob Help

### DISCUSSION THREAD

**Discussion by manujchandra** [qtile/qtile#2742] [Q&A] [answered]
Hello!

By following this [tutorial](https://www.youtube.com/watch?v=rnZm3g_rbwM), I was able to get qtile installed in fedora xfce 34 spin inside a virtualbox VM. The idea is to get it working inside the VM and then use of the production machine.

I have supplied qtile with the [default](https://github.com/qtile/qtile/blob/master/libqtile/resources/default_config.py) configuration.

I have changed the mod key to mod1 because the Virtualbox seems to be doing something with the super key. This is just for testing.

But no matter what I do, I get a config error on the lower left and the keyboard shortcuts are not working. I can change work-spaces by using mouse though. When I press alt+enter or r etc nothing happens.

Could someone guide me on how to configure qtile properly? Thanks.

Edit : I think the config.py is not being read. Because if I change the font to something really large like 20, nothing happens. (even after rebooting)
Edit 2 : I have deleted all the files in the ~/.config/qtile. Still I get a Config Err!.

I installed qtile by using `sudo dnf install qtile`
Version : 0.14.2

![VirtualBox_XFCE_02_09_2021_16_54_55](https://user-images.githubusercontent.com/15191493/131836675-a9dbee98-a144-45cf-a605-abaa4c5e7c05.png)

**Comment by Fabien-Bernard** [comment, marked as answer]
Hello,
The fedora package is not up to date, I doubt it can work with the master default config you linked. You may find something interesting there : #2451
Have you considered installing qtile from source?

**Reply by manujchandra** [reply]
Thanks for the reply. I did not realize that an old version got installed. I think the first logical thing to do would be to try the latest version.
